utils.py

```diff
@@ -185,6 +208,7 @@ def to_ascii(input_string):
 
 
 def dict_from_yaml(yaml_str):
+    # type: (Text) -> Dict[Any, Any]
```

Comment:
Can this be
```suggestion
    # type: (Text) -> Dict[Text, Any]
```

Reply:
Yes it can be as key of YAML is always a string. Done.